packages/x-charts/src/models/seriesType/common.ts

```diff
@@ -58,9 +58,14 @@ export interface CommonSeriesType<TValue> extends SeriesColor<TValue> {
    * There is a default mark type for each series type.
    */
   labelMarkType?: ChartsLabelMarkType;
+  /**
+   * If true, the series will be hidden.
+   * @default false
+   */
+  hidden?: boolean;
```

Comment:
Is it intended to let users provide `hidden` to the series?

If yes, this might be problematic when introducing control on the visibility. You will end-up with two source of information: `series.hidden` and `visibility` (or whatever name you give to the prop)